src/index.html

```diff
@@ -16,7 +32,150 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <!-- #region header -->
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a
+          class="top-bar__logo"
+          href="#"
+        >
+          <img
+            src="./images/themet.png"
+            alt="Base logo"
+          />
+        </a>
+        <div class="top-bar__icons">
+          <a
+            href="#"
+            class="icon icons__icon--phone"
+          ></a>
+          <a
+            href="#menu"
+            class="icon icons__icon--menu"
+          ></a>
+        </div>
+      </div>
+      <div class="header__bottom">
+        <h1 class="title header__title">
+          Welcome
+          <br />
+          to The MET
+        </h1>
+      </div>
+    </header>
+    <!-- #endregion -->
+    <!-- #region aside -->
+    <aside
+      class="menu page__menu"
+      id="menu"
+    >
+      <div class="top-bar menu__top">
+        <a
+          class="top-bar__logo"
+          href="#"
+        >
+          <img
+            src="./images/themet.png"
+            alt="Base logo"
+          />
+        </a>
+        <div class="icons header__icons">
+          <a
+            href="#"
+            class="icon icons__icon--close"
+          ></a>
+        </div>
+      </div>
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Home
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#recommended"
+              >
+                Recommended
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#categories"
+              >
+                Categories
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#how-to-buy"
+              >
+                How to buy
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#contacts"
+              >
+                Contacts
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <a
+          href="tel:+1 234 5555-55-55"
```

Comment:
Wrong phone number in `href="tel:<number>"`

Reply:
Done